tb_plugin/torch_tb_profiler/profiler/loader.py

```diff
@@ -48,11 +50,27 @@ def _parse(self):
                                self.run.name, worker, ex, exc_info=True)
 
     def _process(self):
+        comm_node_lists = []
         for data in self.run.profiles.values():
             logger.debug("Processing profile data")
             data.process()
+            comm_node_lists.append(data.comm_node_list)
             logger.debug("Processing profile data finish")
 
+        worker_num = len(comm_node_lists)
+        for i,_ in enumerate(comm_node_lists[0]):
+            for j,_ in enumerate(comm_node_lists[0][i].kernel_ranges):
```

Comment:
comm_node_lists[0][i] can be replaced with the second tuple returned by enumerate?

Reply:
thanks, done